README.md

```diff
@@ -1,186 +1,94 @@
-</br>
-
-#### 2. Edit the `.envrc` file using your favourite text editor
-
-Replace the secret values (database name, login, password secret key).
-You can get by asking one of the admins of our
-Infrastructure  
-([@arturtamborski](https://github.com/arturtamborski)
-& [@magul](https://github.com/magul))
-and stating your need for it.
-
+#### 2. [Install docker](https://docs.docker.com/engine/install/) to run the app locally on your computer. 
 ```bash
-# edit with `vim`, `code`, or whatever you like :)
-code .envrc
+# Once you install it, go to the location of the repo on your computer and run command: 
+sudo make start 
 ```
+Now you'll be able to access Frontend in you browser tab under the URL: ```localhost:8080``` and push requests to backend on ```localhost:5000```
+(if you don't want to use ```sudo``` follow the instructions [here](https://docs.docker.com/engine/install/linux-postinstall/)).
 
-</br>
-
-#### 3. Load the modified environment variables into your terminal
-
-We've just modified the `.envrc` file with secrets,
-but they are not used by default, we first have to load
-them using `source` command in terminal.  
-There are two ways of doing that.
-
-You can do that manually:
+#### 3. Choose a task from one if the lists below 
+Please leave a comment that you're gonna work on it.
 
+#### 4. Create a new branch for your commits 
+We like to keep master branch clean so:
 ```bash
-# run this in terminal, in the project directory
-source .envrc
+# create a new branch and name it descriptively starting with number of your task e.g. "123_update_something":
+git checkout -b BRANCH NAME
 ```
-
-or you can install [direnv](https://direnv.net/)
-which will load it automatically for you, but this
-program is not required.
-
-</br>
-
-#### 4. Set up SSH tunnel to CodeForPoznan's bastion
-
-SSH config can be obtained the same way as
-secrets - by asking admins of Infrastructure.
-
-In order to set up this SSH tunnel you'll need:
-
-- SSH access to codeforpoznan-bastion (ask admins)
-- SSH config of the connection (ask admins)
-- new terminal window (the tunnel will keep your terminal busy)
-
 ```bash
-# ask admins for config of `.ssh/config` and bastion
-# run this command in new terminal
-ssh codeforpoznan-bastion
+# Then, switch to your new branch that branch with command: 
+git checkout update_something
 ```
+Now you can start developing and commiting your changes!
 
-</br>
-
-#### 5. Restart the environment
-
-This is required in order to load the newly modified
-and sourced environment variables so that the values
-can be used by docker and by backend app.  
-Again, environment variables are not automatically
-passed over on their change, so we need to give it
-a little push for this to work.
-
+#### 5. Push your changes to main repo
+Once you're finished first to commit your changes run three comannds. 
 ```bash
-# run this in terminal, in the project directory
-make stop
-make start
+# First add yout changes to the commit: 
+git add .
```

Comment:
I don't recommend this option. ;-)

Reply:
I wanted to make the simpliest possible rundown. But you're right. I altered it a bit to check the changes first and then add what's important.